WebUI/Pages/ViewModels/EventVM.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using ConestogaInsidersClub.Data.Models;
+
+namespace ConestogaInsidersClub.Pages.ViewModels
+{
+    public class EventVM
+    {
+        public EventVM(Event _event)
+        {
+            EventName = _event.EventName;
+            Description = _event.Description;
+            Location = _event.Location;
+            Capacity = _event.Capacity;
+        }
+        [Required(ErrorMessage = "Event Name is required")]
+        public string EventName { get; set; }
+
+        [Required(ErrorMessage = "Event Description is required")]
+        public string Description { get; set; }
+
+        [Required(ErrorMessage = "Event Location is required")]
+        public string Location { get; set; }
+
+        [Required(ErrorMessage = "Event Capacity is required")]
+        [Range(1, int.MaxValue, ErrorMessage = "Event Capacity should be greater than 0")]
+        public int Capacity { get; set; }
```

Comment:
The admin should not be able to edit the capacity below the number of attendees already registered for the event, right now things like this are possible:
![image](https://user-images.githubusercontent.com/25109066/144261220-42159c65-6ff4-4cc4-b1ae-529f127daad7.png)
This is addressed to @MorganDWalker as well, because the service should also check for that

Reply:
can be done in a follow up PR